Whoaverse/Whoaverse/Views/Subverses/SubverseIndex.cshtml

```diff
@@ -89,8 +92,17 @@
 
             @if (Model.Count < 1)
             {
-                Html.RenderPartial("_DefaultMessage", new ViewDataDictionary() {{"selectedSubverse", selectedSubverse}});
-            }
+                @if (sortingMode == "top")
```

Comment:
The @ character here is not allowed as we're already in a code block and compile fails. Compiler output:  Unexpected "if" keyword after "@" character.  Once inside code, you do not need to prefix constructs like "if" with "@".